src/components/UserCard/index.tsx

```diff
@@ -13,36 +13,46 @@ interface Props {
   user: UserType | null;
   onClick?: () => void;
   optionalTexts?: string[];
+  isFullInfo?: boolean;
 }
 
-const UserCard: React.FC<Props> = ({ user, onClick, optionalTexts }) => {
-  const intl: IntlShape = useIntl();
+const UserCard: React.FC<Props> = ({ user, onClick, isFullInfo }) => {
+  const {formatMessage} = useIntl();
   const localLanguage = useSelector(localLanguageSelector);
-  const format = (id: string): string => intl.formatMessage({ id: id });
 
+  let userInfo : string[];
   const { picture, name, gender, dob, className } = user!;
-  const currentClassName: string = className! + (optionalTexts ? " additional-info" : "");
-  const texts: string[] = [
+  const currentClassName: string = className! + (isFullInfo ? " additional-info" : "");
+  const defaultInfo: string[] = [
     name.first + " " + name.last,
-    format(gender) + ", " + dob.age + " " + format("years old"),
-    format("Date of birth") + ": " + dob.date,
+    formatMessage({id: gender}) + ", " + dob.age + " " + formatMessage({id: "years old"}),
+    formatMessage({id: "Date of birth"}) + ": " + dob.date,
   ];
 
+  switch(isFullInfo){
+    case(true):
```

Comment:
switch/case for boolean condition with two cases ?

Reply:
just to shown that i can use it)